src/main/docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:10
```

Comment:
add maintainer and expose entries

Reply:
1. The MAINTAINER entry is deprecated. Should I use LABEL instead?
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated
2. Who should be added as maintainer (with email)?
3. I will add EXPOSE entry

Reply:
Ok I will send an email and add label addition to ticket scope